service/src/main/kotlin/no/nav/su/se/bakover/service/søknad/lukk/LukkSøknadServiceImpl.kt

```diff
@@ -36,11 +36,19 @@ internal class LukkSøknadServiceImpl(
                 return when (request) {
                     is LukkSøknadRequest.MedBrev -> lukkSøknadMedBrev(request, it)
                     is LukkSøknadRequest.UtenBrev -> lukkSøknadUtenBrev(request, it)
-                }.also {
-                    søknadRepo.hentOppgaveId(søknad.id)?.let { oppgaveId ->
-                        oppgaveClient.lukkOppgave(oppgaveId)
+                }.fold(
```

Comment:
Dersom du har en Either og ønsker å beholde en Either, trenger du ikke bruke fold i dette tilfellet. 

.mapLeft{kunneIkkeLukkeSøknad}
.map{sak ->
                        søknadRepo.hentOppgaveId(søknad.id)?.let { oppgaveId ->
                            oppgaveService.lukkOppgave(oppgaveId).mapLeft {
                                log.warn("Kunne ikke ferdigstille oppgave for ${søknad.id}")
                            }
                        }
                        sak}

Reply:
Siden det ikke gjøres noe nytt i `Left`-caset, kan man også droppe `mapLeft{kunneIkkeLukkeSøknad}` i forslaget der.

Reply:
👍